notebooks/models/Evaluate azure text analytics.ipynb

```diff
@@ -0,0 +1,457 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Evaluate Azure Text Analytics for PII detection using the Presidio Evaluator framework\n",
+    "\n",
+    "Prerequis: \n",
+    " - Azure subscription\n",
+    " - Once you have your Azure subscription, create a Language resource in the Azure portal to get your key and endpoint. After it deploys, click Go to resource.\n",
+    " - You'll need the key and endpoint from the resource you create to connect your application to the API. You'll paste your key and endpoint into the code below later in the quickstart.\n",
+    " - You can use the free pricing tier (Free F0) to try the service, and upgrade later to a paid tier for production.\n",
+    " - To use the Analyze feature, you'll need a Language resource with the standard (S) pricing tier."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
```

Comment:
I think it would be better to have it with no outputs in git, but I don't have a strong preference here.

Reply:
Yes. I removed outputs